acra-censor/acra-censor_configuration_provider.go

```diff
@@ -58,16 +59,23 @@ func (acraCensor *AcraCensor) LoadConfiguration(configuration []byte) ([]QueryHa
 			acraCensor.AddHandler(blacklistHandler)
 			handlers_ = append(handlers_, blacklistHandler)
 			break
-		case LoggerConfigStr:
+		case QueryCaptureConfigStr:
 			if strings.EqualFold(handlerConfiguration.Filepath, "") {
 				break
 			}
-			logger, err := handlers.NewQueryCaptureHandler(handlerConfiguration.Filepath)
+			queryCaptureHandler, err := handlers.NewQueryCaptureHandler(handlerConfiguration.Filepath)
 			if err != nil {
 				return nil, err
 			}
-			acraCensor.AddHandler(logger)
-			handlers_ = append(handlers_, logger)
+			acraCensor.AddHandler(queryCaptureHandler)
+			handlers_ = append(handlers_, queryCaptureHandler)
```

Comment:
do we really need second slice with all handlers? I found one place when it used - in tests. And used to iterate and check length of slice. But in tests we have access to private fields if tests inside the same package. So we don't need this return value, imo